formula/src/main/java/com/instacart/formula/DeferredAction.kt

```diff
@@ -11,11 +11,11 @@ class DeferredAction<Event>(
 ) {
     private var cancelable: Cancelable? = null
 
-    internal var listener: (Event) -> Unit = initial
+    internal var listener: ((Event) -> Unit)? = initial
```

Comment:
Instead of setting empty lambda when disabling this, I made it nullable and set it to null.